pkg/agent/apiserver/handlers/ovsflows/handler.go

```diff
@@ -194,6 +203,20 @@ func HandleFunc(aq agentquerier.AgentQuerier) http.HandlerFunc {
 		namespace := r.URL.Query().Get("namespace")
 		table := r.URL.Query().Get("table")
 		groups := r.URL.Query().Get("groups")
+		tableNamesOnly := r.URL.Query().Get("table-names")
```

Comment:
intuitively, maybe a subresource (`/ovsflows/tablenames`) would make more sense to me
but that may be a better change for not much value
so maybe `/ovsflows?table-names-only` instead? There is no need for a value IMO (`=true`)

Reply:
OK, I was thinking `table-names` is shorter to type. I will remove the `true` value.

Reply:
Done, In current antctl command framework, it doesn't support boolean type of flags. All flags' arguments will be saved into `argMap` which is a type of "map[string]string", so the "true" value is replaced by an empty string.